Week3/homework/2-step3.js

```diff
@@ -1,8 +1,27 @@
 'use strict';
 
-function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const values = [];
-  // Add your code here
-}
+{
+  function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
+    const values = [];
+    for (let i = startIndex; i <= stopIndex; i++) {
+      values.push(i);
+    }
+
+    values.forEach(num => {
+      if (num % 3 === 0)
+        threeCallback(num);
```

Comment:
Make it a habit to always enclose `if-else` and `for` loop blocks in curly braces (`{' and  `}`), even if it only contains one statement. This makes it easy to add more statements to that block later. If you leave out the curly braces it would be easy to make a mistake like this:

```js
if (num % 3 === 0)
  console.log('bla bla');
  threeCallback(num);
if (num % 5 === 0)
  fiveCallback(num);
```

The indentation might lead you to conclude that the first `if` statement block contains two statements. But only the  `console.log` is controlled by the `if` statement.